### DISCUSSION TITLE
Function `getDefaultClassNames` does not exists

### DISCUSSION THREAD

**Discussion by LWJerri** [gpbl/react-day-picker#2346] [Support] [answered]
## Code 

```tsx
import { DayPicker, getDefaultClassNames } from "react-day-picker";

export function MyCalendar() {
  const defaultClassNames = getDefaultClassNames();
  return (
    <DayPicker
      mode="single"
      classNames={{
        today: `border-amber-500`, // Add a border to today's date
        selected: `bg-amber-500 border-amber-500 text-white`, // Highlight the selected day
        calendar: `${defaultClassNames.calendar} shadow-lg p-5` // Add a shadow to the calendar
        chevron: `${defaultClassNames.chevron} fill-amber-500` // Change the color of the chevron
      }}
    />
  );
}
```

## Expected Behavior

The function must be available.

## Actual Behavior

This function does not exists in the imports.

## Screenshots

![image](https://github.com/user-attachments/assets/e9199e84-2f95-4c9c-a926-c40de7b6919f)

**Comment by gpbl** [comment]
@LWJerri are you sure you have installed the latest version? Some cache in your dependency? The function is actually exported: https://daypicker.dev/api/functions/getDefaultClassNames

**Reply by LWJerri** [reply, marked as answer]
> @LWJerri are you sure you have installed the latest version? Some cache in your dependency? The function is actually exported: https://daypicker.dev/api/functions/getDefaultClassNames

Hello. I was on 8.10.1 version, now I on the latest and everything is fine.

**Reply by MichaelzXVI** [reply]
> @LWJerri are you sure you have installed the latest version? Some cache in your dependency? The function is actually exported: https://daypicker.dev/api/functions/getDefaultClassNames

@gpbl 
Hey there. I'm using the latest version (8.10.1) — I encounter the same issue.  `getDefaultClassNames` does not exist in the package at all.

**Comment by gpbl** [comment]
(PS. There's an error in the documentation code that is going to be fixed via your other report #2342)
